museum/sampleRecords/donors.destination.json

```diff
@@ -0,0 +1,578 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "entity_id":"1",
+      "id":"1",
+      "idno":"1",
+      "display_label":"N\/A",
+      "ca_entities.idno":"1",
+      "ca_entities.preferred_labels.displayname":"N\/A",
+      "ca_entities.preferred_labels.surname":"&lt;&gt;"
+    },
+    {
+      "entity_id":"2",
+      "id":"2",
+      "idno":"2",
+      "display_label":"HOPE A.T.",
+      "ca_entities.idno":"2",
+      "ca_entities.preferred_labels.displayname":"HOPE A.T.",
+      "ca_entities.preferred_labels.prefix":"Mrs",
+      "ca_entities.preferred_labels.forename":"A.T.",
+      "ca_entities.preferred_labels.middlename":"A.T.",
+      "ca_entities.preferred_labels.surname":"Hope",
+      "ca_entities.AddressContainer.TownCity":"MT LAWLEY",
+      "ca_entities.LastEditBy":"PENDS",
+      "ca_entities.LastEditDate":"13 November 2014"
+    },
+    {
+      "entity_id":"3",
+      "id":"3",
+      "idno":"4",
+      "display_label":"RILEY",
+      "ca_entities.idno":"4",
+      "ca_entities.preferred_labels.displayname":"RILEY",
+      "ca_entities.preferred_labels.prefix":"Mrs",
+      "ca_entities.preferred_labels.middlename":"O.L.",
+      "ca_entities.preferred_labels.surname":"Riley",
+      "ca_entities.AddressContainer.Address1":"Bishops House",
+      "ca_entities.AddressContainer.TownCity":"PERTH",
+      "ca_entities.AddressContainer.State":"WA",
+      "ca_entities.LastEditBy":"LesB",
+      "ca_entities.LastEditDate":"28 June 2006"
+    },
+    {
+      "entity_id":"4",
+      "id":"4",
+      "idno":"5",
+      "display_label":"HUTT, Mrs",
+      "ca_entities.idno":"5",
+      "ca_entities.preferred_labels.displayname":"HUTT, Mrs",
+      "ca_entities.preferred_labels.prefix":"Mrs",
+      "ca_entities.preferred_labels.forename":"Mrs",
+      "ca_entities.preferred_labels.middlename":"MRS",
+      "ca_entities.preferred_labels.surname":"Hutt,",
+      "ca_entities.AddressContainer.Address1":"28 Thomas Street",
+      "ca_entities.AddressContainer.TownCity":"NEDLANDS",
+      "ca_entities.AddressContainer.State":"WA",
+      "ca_entities.AddressContainer.PostCode":"6009",
+      "ca_entities.DonorNotes":"Donated in early 1978",
+      "ca_entities.LastEditBy":"IST",
+      "ca_entities.LastEditDate":"19 December 2000"
+    },
+    {
+      "entity_id":"5",
+      "id":"5",
+      "idno":"6",
+      "display_label":"FAIRBAIRN, Miss, and Dr Fairbairn",
+      "ca_entities.idno":"6",
+      "ca_entities.preferred_labels.displayname":"FAIRBAIRN, Miss, and Dr Fairbairn",
+      "ca_entities.preferred_labels.prefix":"Miss",
+      "ca_entities.preferred_labels.forename":"Miss, and Dr Fairbairn",
```

Comment:
This is the source data, and it is possible to change the entity type of this entity to a 'FAMILY' which then has simpler labels, or add an additional name to the entity, or have two entities. but I think just keep it as they have been using it.